<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDialog</class>
 <widget class="QDialog" name="RoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>642</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Room</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background:url(:/images/background_main.png)
}
#startButton{
	background:url(:/images/btn_letsplay_room.png) transparent no-repeat;
	border:none;
}
#startButton:hover{
	background:url(:/images/btn_letsplay_room_hover.png) transparent no-repeat;
	border:none;
}
#startButton:pressed{
	background:url(:/images/btn_letsplay_room_hover.png) transparent no-repeat;
	border:none;
}
#invite{
	background:url(:/images/btn_invite.png) transparent no-repeat;
	border:none;
}
#invite:hover{
	background:url(:/images/btn_invite_hover.png) transparent no-repeat;
	border:none;
}
#invite:pressed{
	background:url(:/images/btn_invite.png) transparent no-repeat;
	border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="playersVBox">
     <item>
      <widget class="QPushButton" name="invite">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>31</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>31</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton{
	background:url(:/images/btn_kick.png) transparent no-repeat;
	border:none;
}
#pushButton:hover{
	background:url(:/images/btn_kick_hover.png) transparent no-repeat;
	border:none;
}
#pushButton:pressed{
	background:url(:/images/btn_kick_hover.png) transparent no-repeat;
	border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="ConversationSet" name="convSet" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConversationSet</class>
   <extends>QWidget</extends>
   <header location="global">conversationset.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
